x-pack/plugins/observability_ai_assistant/public/components/feedback_buttons.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButtonEmpty, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+
+export type Feedback = 'positive' | 'negative';
+
+interface FeedbackButtonsProps {
+  onClickFeedback: (feedback: Feedback) => void;
+}
+
+export function FeedbackButtons({ onClickFeedback }: FeedbackButtonsProps) {
+  return (
+    <EuiFlexGroup responsive={false} direction="row" alignItems="center" gutterSize="s">
+      <EuiFlexItem grow={false}>
+        <EuiText color="subdued" size="s">
+          <em>
+            {i18n.translate('xpack.obsAiAssistant.insight.feedbackButtons.title', {
```

Comment:
this should be `xpack.observabilityAiAssistant` (as configured in x-pack/.i18nrc.json)

Reply:
Updated